docs/client.md

```diff
@@ -202,13 +213,13 @@ You will then need to set the various fields as appropriate. If you are running
 
 ##### eik.json properties
 
-| property     | description                                                         |
-| ------------ | ------------------------------------------------------------------- |
-| name         | App name, must be unique to the Eik server                          |
-| server       | Address to the asset server                                         |
-| js           | Configuration for JavaScript assets                                 |
-| css          | Configuration for CSS assets                                        |
-| import-map   | Specify import maps to be used to map bare imports during bundling  |
+| property   | description                                                        |
+| ---------- | ------------------------------------------------------------------ |
+| name       | App name, must be unique to the Eik server                         |
+| server     | Address to the asset server                                        |
+| js         | Configuration for JavaScript assets                                |
+| css        | Configuration for CSS assets                                       |
+| import-map | Specify import maps to be used to map bare imports during bundling |
```

Comment:
Looks like this section is out of date. Should be a files property and no js/css etc.

Reply:
It seems like this `client.md` file is not rendered in the sidebar, so it's not shown anywhere unless you know the URL.